examples/Clients/Chatter/Program.cs

```diff
@@ -39,10 +43,8 @@ static async Task<int> Main(string[] args)
             var name = args[0];
 
             // Server will only support Https on Windows and Linux
-            var credentials = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? ChannelCredentials.Insecure : ClientResources.SslCredentials;
-
-            var channel = new Channel("localhost:50051", credentials);
-            var client = new Chatter.ChatterClient(channel);
+            var certificate = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? null : new X509Certificate2(Path.Combine(Resources.CertDir, "client.crt"));
+            var client = GrpcClientFactory.Create<Chatter.ChatterClient>(@"https://localhost:50051", certificate);
```

Comment:
Does it make sense to replace existing examples (that are fully working) with examples that use Grpc.NetCore.HttpClient and miss important functionality (trailers)?
It sounds more like we should fork one of the clients and create a separate example for Grpc.NetCore.HttpClient.

Same for other examples.

Reply:
I've reverted the clients to use the c-core client.